data/json/items/containers.json

```diff
@@ -566,12 +566,13 @@
     "material": "aluminum",
     "symbol": ")",
     "color": "light_blue",
+    "qualities": [ [ "CONTAIN", 1 ], [ "BOIL", 2 ] ],
     "pocket_data": [
       {
         "pocket_type": "CONTAINER",
         "rigid": true,
         "watertight": true,
-        "open_container": true,
+        "open_container": false,
```

Comment:
I think we got off on the wrong page again - open_container is supposed to be here, but it's _also_ supposed to be overridden by the `_sealed` variable. if you do this, you can store things in the cans no problem, but the cans not being sealed is still an issue.

Reply:
I don't follow. To what `_sealed` variable are you referring?

Reply:
item_pocket::_sealed

Reply:
if item_pocket::sealed() == true then it is not an open container, but if it is false, it checks the pocket data for this bool on whether or not it spills if you put it in a container.

Reply:
OK, I think I understand - because these containers do not have `sealed_data`, they are considered `resealable()==true`, and thus always `sealed()==false`.

So I suppose the solution here is to add `sealed_data` to these containers. I will try that.

Reply:
yes exactly. i didn't realize they didn't have it.

Reply:
OK, I think I'm even more confused now. These items do have `sealed_data`, thus `resealable()` should be false, and thus `sealed()` should be true.

I don't understand what change you are asking me to make.

Reply:
well that means there's a bug somewhere in the C++, somewhere either is unsealing it too soon ( my suspicion) or ignoring the sealed data